testing/trino-testing/src/main/java/io/trino/testing/AbstractTestEngineOnlyQueries.java

```diff
@@ -5315,47 +5315,37 @@ public void testShowSession()
     public void testSetSession()
     {
         MaterializedResult result = computeActual("SET SESSION test_string = 'bar'");
-        assertTrue((Boolean) getOnlyElement(result).getField(0));
```

Comment:
maybe instead of removing, replace with assertion that there are no fields in the result

```java
assertThat(result.getMaterializedRows()).isEmpty();
assertThat(result.getTypes()).isEmpty(); // i.e. no columns
```

this can be wrapped in a method like `assertNoRelationalResult`